### ISSUE TITLE
Getting audio only from earpiece speaker in iOS

### ISSUE THREAD

**Issue by ItsAman0P** [TheWidlarzGroup/react-native-video#3048]
## Bug

I have used this library to preview audio and video messages in my chat app.
In android the preview is working fine and getting audio from both the speaker.
But in iOS, I'm getting audio from only one speaker, from the earpiece. 

## Platform
iOS 15 
iPhone 13 Pro Max and iPhone 6s Plus

Which player are you experiencing the problem on:
* iOS

## Environment info

React native info output:

```bash
 System:
    OS: macOS 11.7.1
    CPU: (4) x64 Intel(R) Core(TM) i5-5575R CPU @ 2.80GHz
    Memory: 87.00 MB / 8.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.16.0 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 8.13.2 - ~/Documents/hello-mobile/node_modules/.bin/npm
    Watchman: 2022.11.28.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.2, iOS 15.2, macOS 12.1, tvOS 15.2, watchOS 8.3
    Android SDK: Not Found
  IDEs:
    Android Studio: 2021.3 AI-213.7172.25.2113.9123335
    Xcode: 13.2.1/13C100 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.17 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.1.0 => 18.1.0 
    react-native: 0.70.6 => 0.70.6 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

Library version: 5.2.1 and 6.0.0-alpha.5

## Steps To Reproduce

1. Install the package.
2. Pass a video url to source uri in Video component.

**Comment by BraveEvidence** [comment]
This will help https://www.youtube.com/watch?v=vVI7ZAZq5e0

**Comment by codal-mpawar** [comment]
Facing the same issue. video player not switch to the external speakers.

**Comment by freeboub** [comment]
I think you need this PR ? https://github.com/react-native-video/react-native-video/pull/2888

**Comment by KrzysztofMoch** [comment]
Issue is fixed in 6.0.0-beta.0 - I tested it on real device (iOS 17.2)
